crates/aptos-rosetta/src/client.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::{AccountBalanceRequest, AccountBalanceResponse};
+use aptos_rest_client::aptos_api_types::mime_types::JSON;
+use aptos_types::{account_address::AccountAddress, chain_id::ChainId};
+use reqwest::{header::CONTENT_TYPE, Client as ReqwestClient};
+use url::Url;
+
+pub struct RosettaClient {
+    address: Url,
+    inner: ReqwestClient,
+}
+
+impl RosettaClient {
+    pub fn new(address: Url) -> RosettaClient {
+        RosettaClient {
+            address,
+            inner: ReqwestClient::new(),
+        }
+    }
+
+    pub async fn account_balance_simple(
+        &self,
+        account: AccountAddress,
+        chain_id: ChainId,
+    ) -> anyhow::Result<AccountBalanceResponse> {
+        let request = AccountBalanceRequest {
+            network_identifier: chain_id.into(),
+            account_identifier: account.into(),
+            block_identifier: None,
+            currencies: None,
+        };
+        self.account_balance(&request).await
+    }
+
+    pub async fn account_balance(
```

Comment:
What is the difference between this and account_balance_simple?

Reply:
'account_balance' is exactly what the spec requires for Rosetta.  'account_balance_simple' let's me skip the serialization and do it all in one place.

Think of it as a SDK on a spec, with an extension